examples/tree_1d_fdsbp/elixir_advection_upwind_periodic.jl

```diff
@@ -0,0 +1,57 @@
+# !!! warning "Experimental implementation (upwind SBP)"
+#     This is an experimental feature and may change in future releases.
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the linear scalar advection equation equation
+
+equations = LinearScalarAdvectionEquation1D(1.0)
+
+function initial_condition_sin(x, t, equation::LinearScalarAdvectionEquation1D)
+    return SVector(sinpi(x[1] - equations.advection_velocity[1] * t))
+end
+
+D_upw = upwind_operators(SummationByPartsOperators.periodic_derivative_operator,
+                         accuracy_order = 4,
+                         xmin = -1.0, xmax = 1.0,
+                         N = 64)
+flux_splitting = splitting_lax_friedrichs
+solver = FDSBP(D_upw,
+               surface_integral = SurfaceIntegralUpwind(flux_splitting),
+               volume_integral = VolumeIntegralUpwind(flux_splitting))
```

Comment:
Should this be
```suggestion
solver = PeriodicFDSBP(D_upw,
                       surface_integral = SurfaceIntegralUpwind(flux_splitting),
                       volume_integral = VolumeIntegralUpwind(flux_splitting))
```
to be consistent with the dispatch type used internally?

Reply:
No, I would like to keep `PeriodicFDSBP` as a purely internal name without exporting it. It's just a way for me to make dispatch easier to read. That's how we handle the situation with FD SBP operators wrapped in `DGMulti`, too.

Reply:
Sounds reasonable. Maybe add a short comment to the definition of it, e.g.,
```julia
# Internal type for easier-to-read dispatch (not exported)
PeriodicFDSBP = ...
```
?

Reply:
Done in 2bfcb1a